src/assets/lima-config.yaml

```diff
@@ -14,6 +14,11 @@ provision:
   script: |
     #!/bin/sh
     hostname lima-rancher-desktop
+- # Create host.rancher-desktop.internal and host.docker.internal aliases for host.lima.internal
+  mode: system
+  script: |
+    #!/bin/sh
+    sed -i 's/host.lima.internal.*/host.lima.internal host.rancher-desktop.internal host.docker.internal/' /etc/hosts
```

Comment:
> Should we capture the rest of the line in a \(.*\) group and echo it out?

No, I'm intentionally discarding it. The changes we make are persisted, so we would append the additional names over and over on each reboot otherwise.

If Lima ever adds more than one name to that line, then we have to adapt. But most likely Lima will generate additional lines if/when it wants to define more aliases for the gateway address...